libs/partners/zenguard/tests/integration_tests/test_tools.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import pytest
+
+from langchain_zenguard import Detector, ZenGuardTool
+
+
+@pytest.fixture()
+def zenguard_tool():
+    if os.getenv("ZENGUARD_API_KEY") is None:
+        raise ValueError("")
+    return ZenGuardTool()
```

Comment:
we should probably hardcode the mock key, otherwise this is going to fail the unit tests. Since nobody is going to set our key in CI

Reply:
They have this in their documentation:
Add secret names to manual integrations workflow in .github/workflows/_integration_test.yml

Reply:
How do you provide the actual key to  .github/workflows/_integration_test.yml